ecr-login/config/log.go

```diff
@@ -27,14 +27,19 @@ func SetupLogger() {
 }
 
 func logrusConfig() {
-	logfile, err := homedir.Expand(GetCacheDir() + "/log/ecr-login.log")
+	logdir, err := homedir.Expand(GetCacheDir() + "/log")
 	if err != nil {
-		fmt.Errorf("%v", err)
-		logfile = "/tmp/.ecr/log/ecr-login.log"
+		fmt.Errorf("log: failed to find directory: %v", err)
```

Comment:
The result of `fmt.Errorf` not used?

Reply:
...thank you for catching my brain fart.  This should be `fmt.Sprintf(os.Stderr, ...)`.  Fix incoming...

Reply:
Done